sdk/core/azure-core/test/ut/environment_log_level_listener_test.cpp

```diff
@@ -119,17 +108,17 @@ TEST_F(EnvironmentLogLevelListenerTest, LogLevelVerbose)
 {
   EnvironmentLogLevelListener::SetInitialized(false);
   SetLogLevel("verbose");
-  auto level = EnvironmentLogLevelListener::GetLogLevel(Logger::Level::Verbose);
+  auto level = EnvironmentLogLevelListener::GetLogLevel(Logger::Level::Error);
   EXPECT_EQ(level, Logger::Level::Verbose);
 
   EnvironmentLogLevelListener::SetInitialized(false);
   SetLogLevel("debug");
-  level = EnvironmentLogLevelListener::GetLogLevel(Logger::Level::Verbose);
```

Comment:
Is there a reason to change the log levels here ?

Reply:
Yes - so that what we set is not the value that we end up getting, meaning more chance that code works. Otherwise, if logger does not end up being created because it wrongfully thinks that the corresponding value was not being set, you get the parameter value, which was `Verbose` before. Now, it is `Error`, so if logger is not getting created, the test will get `Error` and not `Verbose`. Otherwise, we wouldn't being able to distinguish between correct work and not working.